pkg/virtualKubelet/liqoNodeProvider/reconciler.go

```diff
@@ -276,12 +329,36 @@ func (p *LiqoNodeProvider) updateNode() error {
 }
 
 func (p *LiqoNodeProvider) handleResourceOfferDelete(resourceOffer *sharingv1alpha1.ResourceOffer) error {
+	ctx := context.TODO()
+
+	if err := p.cordonNode(ctx); err != nil {
+		klog.Errorf("error cordoning node: %v", err)
+		return err
+	}
+
+	if err := p.drainNode(ctx); err != nil {
```

Comment:
Is it fine to remain blocked here for long periods of time, or would it cause the loss of potential updates (e.g. about the status?)

Reply:
Yes, we will lose potential updates, but they are updates on resources/conditions over a node that we are deleting and will not exists anymore with this VK instance

Reply:
I agree in the general case, but I am just a bit worried whether this would apply also in case of errors/problems.